pallets/eth-bridge/src/lib.rs

```diff
@@ -894,7 +894,7 @@ pub mod pallet {
                     },
                 Call::submit_ethereum_events { author, events_partition, signature } =>
                     if AVN::<T>::signature_is_valid(
-                        &encode_eth_event_submission_data(
+                        &(
```

Comment:
why are we removing the function call? Won't this potentially cause a mistmatch?

Reply:
its a missmatch now. Validation expects a tupple, not encoded data.